Included StoneCo (#782)
remoteintech/remote-jobs: company-profiles/stoneco.md

# StoneCo

## Company blurb

[StoneCo](https://www.stone.co/) is a leading provider of financial technology solutions that empower merchants and integrated partners to conduct electronic commerce seamlessly across in-store, online, and mobile channels. Stone has developed a deep client-centric culture that seeks to delight our clients rather than to simply provide them with a solution or service. Stone serves clients of all sizes and types that transact online, offline or have an omni-channel sales approach. Stone also serves many integrated partners, which use or embed Stone’s solutions into their own offerings to enable their customers to conduct commerce more conveniently.

Our purpose is clear: changing the way the payment system works. To do so, we work hard to provide an equal playing field for entrepreneurs.

## Company size

> 4000+ employees and constantly growing to innovate and bring solutions to our reason: the customer.

Reviewer: More clear since a "reason" is always a reason "for" something:
```suggestion
4000+ employees and constantly growing to innovate and bring solutions to our reason for existing: the customer.
```

## Remote status

> For many teams we offer full remote options, but you can work either in offices around Brazil as partially remote depending from your team and position.

Reviewer: >you can work either in offices around Brazil as partially remote depending from your team and position.
This isn't very clear and there are a few errors with prepositions.
Maybe rephrase like this?
```suggestion
For many teams we offer fully remote options, and for the rest you can work either in offices around Brazil or partially remote depending on your team and position.
```

## Region

Brazil

## Company technologies

> Tools and technologies at StoneCo generally is decided by each squad and varies according the solution. Depending from your squad, you can work with: C#, Elixir, Kubernetes, Docker, RabbitMQ, GCP, Azure, Ansible, Terraform and much more. You can check the job description to know more about a specific stack.

Reviewer: Grammar fixes:
```suggestion
Tools and technologies at StoneCo generally are decided by each squad and vary according to the solutions needed. Depending on your squad, you can work with: C#, Elixir, Kubernetes, Docker, RabbitMQ, GCP, Azure, Ansible, Terraform and many more. You can check the job description to know more about a specific stack.
```

## Office locations

> The main offices is located at São Paulo and Rio de Janeiro, but we has more around Brazil.

Reviewer: Grammar fixes:
```suggestion
The main offices are located in São Paulo and Rio de Janeiro, but we have more around Brazil.
```
